testing_exercise/testing.js

```diff
@@ -0,0 +1,38 @@
+function replaceWith(string, pattern, replacer) {
+  var newString = [];
+  string.split('').forEach(function(letter) {
+    if (letter.toLowerCase() === pattern.toLowerCase()) {
```

Comment:
function should be case *sensitive*, not case *insensitive*. (you'll need to update one of your tests as well)